Code/Framework/AzCore/AzCore/Settings/SettingsRegistryMergeUtils.cpp

```diff
@@ -337,16 +335,17 @@ namespace AZ::SettingsRegistryMergeUtils
             return engineRoot;
         }
 
-        // Step 3 locate the project root and attempt to find the engine root using the registered engine
-        // for the project in the project.json file
+        // Step 3 locate the project root and attempt to find the engine using the engine_path
+        // in user/project_settings.json or find the first registered engine with the same name
+        // as the one in project.json
         AZ::IO::FixedMaxPath projectRoot;
         settingsRegistry.Get(projectRoot.Native(), FilePathKey_ProjectPath);
         if (projectRoot.empty())
         {
             return {};
         }
 
-        // Use the project.json and engine manifest to locate the engine root.
+        // Use user/project_settings.json, project.json and engine manifest to locate the engine root.
```

Comment:
This comment should be updated to be `user/project.json`
```suggestion
        // Use <project-root>/user/project.json, project.json and engine manifest to locate the engine root.
```